src/components/player/hls-source.tsx

```diff
@@ -0,0 +1,31 @@
+import React, {Component} from 'react'
+import Hls from 'hls.js'
```

Comment:
I was having compatibility issues with this and the `cueplayer-react` project so I moved it here to "user land" as they demonstrate